src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -1,4 +1,5 @@
 (function(global, doc, $, React, ReactDOM, eZ, Routing) {
+    const SELECTOR_BULK_OPERATION_FAILED_MODAL = '.ez-bulk-operation-failed-modal';
```

Comment:
`SELECTOR_MODAL_BULK_ACTION_FAIL`

Reply:
I don't like this name: `.ez-bulk-operation-failed-modal`

Reply:
So you want to change all names `bulk-operation-failed` to `bulk-action-failed`?

Reply:
Yes, it's shorter but still there's something in the class name I don't like. Can't it be even shorter by using different words describing the same element?

Reply:
Unfortunately, I was unable to come up with shorter name. 😕